go/libkb/errors.go

```diff
@@ -2040,3 +2041,15 @@ type LoginStateTimeoutError struct {
 func (e LoginStateTimeoutError) Error() string {
 	return fmt.Sprintf("LoginState request timeout - attempted: %s, active request: %s, duration: %s", e.ActiveRequest, e.AttemptedRequest, e.Duration)
 }
+
+type KBFSNotRunning struct{}
```

Comment:
Can we rename this to have an `Error` suffix to be consistent with every other error type in this file?